reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/OnErrorResumeNext.kt

```diff
@@ -0,0 +1,35 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.base.CompleteCallback
+import com.badoo.reaktive.base.ValueCallback
+import com.badoo.reaktive.base.subscribeSafe
+import com.badoo.reaktive.base.tryCatch
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+
+fun <T> Observable<T>.onErrorResumeNext(errorResumeNextSupplier: (Throwable) -> Observable<T>): Observable<T> =
```

Comment:
Let's also add `onErrorResumeNext(next: Observable)` that will just use this function.